chore: Add the 2.8.4.3 Gateway changelog (#6029)
Kong/docs.konghq.com: app/_hub/kong-inc/aws-lambda/overview/_index.md

{% endif_plugin_version %}

{% if_plugin_version eq:2.8.x %}

> The AWS-Lambda plugin will automatically fetch the IAM role credential according to the following

Reviewer: ```suggestion
The AWS Lambda plugin automatically fetches the IAM role credential according to the following
```

precedence order:

> - Fetch from credentials defined in environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
> - Fetch from profile and credential file, defined by `AWS_PROFILE` and `AWS_SHARED_CREDENTIALS_FILE`.
> - Fetch from ECS [container credential provider](https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html).
> - Fetch from EKS [IAM roles for service account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
> - Fetch from EC2 IMDS metadata.(Both v1 and v2 are supported)

Reviewer: ```suggestion
1. Fetch from credentials defined in environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
2. Fetch from profile and credential files, defined by `AWS_PROFILE` and `AWS_SHARED_CREDENTIALS_FILE`.
3. Fetch from ECS [container credential provider](https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html).
4. Fetch from EKS [IAM roles for service account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
5. Fetch from EC2 IMDS metadata (both v1 and v2 are supported).
```

> What's more, if you also provide the `aws_assume_role_arn` option, the plugin will try to perform

Reviewer: ```suggestion
If you also provide the `aws_assume_role_arn` option, the plugin tries to perform
```

an additional [AssumeRole](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)

> action, which requires the Kong process to make HTTPS request to AWS STS service API, after
> configuring AWS access key/secret or fetching credentials automatically from EC2/ECS/EKS IAM roles.

Reviewer: ```suggestion
action. The AssumeRole action requires the {{site.base_gateway}} process to make an HTTPS request to the AWS STS service API after
configuring AWS access key/secret or fetching credentials automatically from EC2/ECS/EKS IAM roles.
```

> If it succeeds, the plugin will fetch a temporary security credentials that represents

Reviewer: ```suggestion
If it succeeds, the plugin fetches temporary security credentials that represent
```

> that the plugin now has the access permission configured in the target assumed role. Then the plugin will try to invoke lambda function based on the target assumed role.

Reviewer: ```suggestion
that the plugin now has the access permission configured in the target assumed role. Then, the plugin will try to invoke the lambda function based on the target assumed role.
```
